src/olympia/abuse/cron.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime, timedelta
+
+from django.template import loader
+
+from olympia.amo.utils import send_mail
+
+from .models import AbuseReport, CinderJob
+
+
+def reports_without_cinder_id_qs():
+    """check for any abuse reports that have been created more than an hour ago and
+    don’t have a cinder job yet"""
+    one_hour_ago = datetime.now() - timedelta(hours=1)
+    return AbuseReport.objects.filter(
+        reason__in=tuple(AbuseReport.REASONS.REPORTABLE_REASONS.values),
+        cinder_job_id__isnull=True,
+        created__lt=one_hour_ago,
+    )
+
+
+def unresolved_cinder_handled_jobs_qs():
+    """check for any abuse reports created more than Y business days ago that have a
+    cinder job ID but don’t have a cinder decision."""
+    sla_date = datetime.now() - timedelta(days=3)
+    return (
+        CinderJob.objects.unresolved()
+        .exclude(
+            id__in=CinderJob.objects.resolvable_in_reviewer_tools().values_list(
+                'id', flat=True
+            )
+        )
```

Comment:
there's a more efficient way to do this, but it involves rewriting `resolvable_in_reviewer_tools` to use `Q`s and being able to pass a negation arg to get cases that _aren't_ resolvable in the reviewer tools, i.e. are resolvable in cinder instead.